File: Assignment_13/Assignment3.c
#include<stdio.h>

void Pattern(int iRow, int iCol)
{
    int i = 0, j = 0;
    char ch = 'A';

    for(i = 1, ch = 'A'; i <= iRow; i++, ch++)
    {
        for(j = 1; j <= iCol; j++)
        {
            printf("%c\t",ch);
        }
        printf("\n");
    }
}

int main()
{   
    int iValue1 = 0;
    int iValue2 = 0;

    printf("Enter Numbers Of Rows :");
    scanf("%d",&iValue1);

    printf("Enter Numbers Of Colums :");
    scanf("%d",&iValue2);

    Pattern(iValue1,iValue2);

    return 0;
}

File: Assignment____45/Assignment3.java
import java.util.*;

class ReverseColums
{
    public void ReverseColums(int Arr[][],int Row, int Col)
    {
        for(int j = 0; j < Col; j++)
        {
            int start = 0;
            int end = Row - 1;

            while(start < end)
            {
                int temp = Arr[start][j];
                Arr[start][j] = Arr[end][j];
                Arr[end][j] = temp;

                start++;
                end--;
            }
        }
    }
}
class Assignment3
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the Rows :");
        int iRow = sobj.nextInt();

        System.out.println("Enter the Colums :");
        int iCol = sobj.nextInt();

        int Arr[][] = new int [iRow][iCol];
        System.out.println("Enter the Marix :");
        for(int i = 0; i < iRow; i++)
        {
            for(int j = 0; j < iCol; j++)
            {
                Arr[i][j] = sobj.nextInt();
            }
        }
        System.out.println("Matrix Elements is :");
        for(int i = 0; i < iRow; i++)
        {
            for(int j = 0; j < iCol; j++)
            {
                System.out.print(Arr[i][j] +"\t");
            }
            System.out.println();
        }

        ReverseColums robj  = new ReverseColums();
        robj.ReverseColums(Arr,iRow,iCol);

        System.out.println("\nMatrix After Reversing Columns:");
        for(int i = 0; i < iRow; i++)
        {
            for(int j = 0; j < iCol; j++)
            {
                System.out.print(Arr[i][j] + "\t");
            }
            System.out.println();
        }

        sobj.close();

    }
}